blackbox/docs/sql/administration/ingestion_rules.txt

```diff
@@ -0,0 +1,97 @@
+.. highlight:: psql
+.. _administration-ingestion-rules:
+
+===============
+Ingestion Rules
+===============
+
+Introduction
+============
+
+In CrateDB ingestion rules are meant to be used together with an ingestion
+implementation, for the purpose of filtering data streams and storing them in
+the corresponding target tables.
+
+What Is An Ingestion Rule ?
+===========================
+
+Ingestion rules serve as a configuration for ingestion implementations.
+
+An ingestion rule is defined by a ``rule_name``, a ``source_ident`` which
+is a unique identifier of the ingestion implementation and a ``condition``
+which will serve as a filter for the data and finally a target ``table_ident``
+where the filtered data will be stored.
+
+Creating and Deleting Ingestion Rules
+=====================================
+
+Supposing we have installed an ingestion implementation for an ``mqtt`` broker,
+and we would like to filter temperature data streams and store them to
+``temp_data.raw``::
+
+    CREATE INGEST RULE temperature
+    ON mqtt
+    WHERE topic like 'temperature/%'
+    INTO temp_data.raw;
+
+It is possible to create and drop ingestion rules even if the corresponding
+ingestion implementation is not available. However, the target table
+needs to be created manually before, and it needs to have the same columns
```

Comment:
" However, the target table needs to be created manually before" - can we complete this with "... manually before the ingestion implementation becomes available and it needs to have the same structure as the ingestion implementation's data structure"